### ISSUE TITLE
Deprecation warning:  CLI options definitions were upgraded with "type" property

### ISSUE THREAD

**Issue by sc0ttdav3y** [serverless/safeguards-plugin#23]
Hi,

Thanks for this plugin — it looks like just what I need. I've just installed it, and when it runs I get the following deprecation warning from Serverless:

```
Serverless: Deprecation warning: CLI options definitions were upgraded with "type" property (which could be one of "string", "boolean", "multiple"). Below listed plugins do not predefine type for introduced options:
             - ServerlessSafeguardPlugin for "org", "app", "service", "policy-file"
            Please report this issue in plugin issue tracker.
            Starting with next major release, this will be communicated with a thrown error.
            More Info: https://www.serverless.com/framework/docs/deprecations/#CLI_OPTIONS_SCHEMA
```

My environment:

```
Operating System:          darwin
Node Version:              14.4.0
Framework Version:         2.39.1 (standalone)
Plugin Version:            4.5.3
SDK Version:               4.2.2
Components Version:        3.9.1
```

**Comment by pgrzesik** [comment]
Thanks for reporting @sc0ttdav3y :bow: It will be fixed soon with next version of the plugin.

**Comment by pgrzesik** [comment]
Fix was released with `1.2.0` version of the plugin :+1:
